R/run_prep_calculation_loans.R

```diff
@@ -1,13 +1,20 @@
 #' Run stress testing data preparation for loans
 #'
+#'
 #' @inheritParams run_stress_test
+#' @param data_prep_output_path Path where results are written.
+#'   NOTE: This is a workflow that is needed exclusively in preparation of
+#'   [run_stress_test()] for loan books. It creates input data for the stress
+#'   test. A recommended setting is to set `data_prep_output_path` to to the same
```

Comment:
Is it just recommended? If I were to set the output of the prep somewhere else, I could still get the main calc to run, by changing the specific path?

Reply:
I do not entirely understand the question. I would say it is recommended, because that is easiest, but not enforced, because a user can pick any path, they will jsut have to do some copy paste action then

Reply:
okay that's what I meant. Got it, I will strongly suggest it to be used that way in any meeting/document